css/main.css

```
@@ -8,3 +8,84 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+.beyonce{
+
+
+}
+
+.perfilbeyonce{
+width: 277px;
+height: 142px;
+position: absolute;
+background-image: url('../img/beyonce.jpg');
+margin-bottom: 2px;
+}
+
+.opacidade{
+  background: #2e2e2e;;
+ width: 277px;
+height: 142px;
+position: absolute;
+opacity: 0.7;
+margin-bottom: 2px;
+
+}
+.boxtexto{
+  width: 277px;
+  height: 142px;
+  position: relative;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.85;
+  letter-spacing: normal;
+  justify-content:center;
+  
+  color: #ffffff;
+  
+  display: flex;
+  flex-direction: column;
+  align-items:center;
+
+ 
+  
+}
+
+.title{
+  font-size: 18px;
+  
+
+}
+.music{
+  font-size: 13px;
+}
+
+
+.perfilkaty{
+  width: 277px;
+  height: 142px;
+  background-image: url('../img/katy.jpg');
+  background-size: cover;
+  position: absolute;
+}
+
+.perfilrida{
```

Separar o nomes com traços, tipo:

```suggestion
.perfil-rida{
```